### PR TITLE
Fix vbproj build in VS and update configurations

### PR DESCRIPTION
**Pull request by danmosemsft**
The only .vbproj in CoreFX does not build in VS because it is using `/sdkpath:C:\git\corefx\bin/ref/netcoreapp/` in VS instead of `/sdkpath:C:\git\corefx\bin/ref/netstandard/`. 
That is caused by not having a netstandard configuration, which seems to be because for .vbproj the usual pattern like `<PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard-Debug|AnyCPU'" />` isn't sufficient: you have to also (or instead?) add it to `<Configurations>`

On the command line, configurations are chosen by other means.

After this change, it builds clean without errors.

### CODE REVIEW

**Comment by paul1956** [comment]
@danmosemsft closer, I added


```
<Configurations>netstandard-Debug;netstandard-Release;netstandard1.3-Debug;netstandard1.3-Release</Configurations>
```
but now get
```
Severity	Code	Description	Project	File	Line	Suppression State
Error		Could not find a configuration for ProjectReference 'C:\Users\PaulM\Source\Repos\corefx\src\/Microsoft.VisualBasic/ref/Microsoft.VisualBasic.csproj' from configurations 'netstandard' when building 'Microsoft.VisualBasic' for configuration 'netstandard1.3-Debug' .	Microsoft.VisualBasic (src\Microsoft.VisualBasic)	C:\Users\PaulM\Source\Repos\corefx\buildvertical.targets	168	
```

**Comment by danmosemsft** [comment]
@paul1956 doh! Can you go to the configuration manager and check that this project configuration is `netstandard-Debug` not `netstandard1.3-Debug`? 

If that doesn't do it, what if you close VS, remove `netstandard1.3-Debug;netstandard1.3-Release` and then reopen VS?

**Comment by danmosemsft** [comment]
Mine looks like this

image

**Comment by paul1956** [comment]
image
When I match yours, VS seems to work.

**Comment by paul1956** [comment]
The program now compiles with VS with the 2 errors below and no squiggly errors, but the build window has lots of expected errors. Is this expected? How do I get VS to show the issues?

image

File: src/Microsoft.VisualBasic/src/Microsoft.VisualBasic.vbproj [hunk 1]
```diff
@@ -13,6 +13,7 @@
     <AssemblyName>Microsoft.VisualBasic</AssemblyName>
     <NoStdLib>true</NoStdLib>
     <RootNamespace />
+    <Configurations>netstandard-Debug;netstandard-Release;netstandard1.3-Debug;netstandard1.3-Release</Configurations>
```
Comment (eerhardt) [inline src/Microsoft.VisualBasic/src/Microsoft.VisualBasic.vbproj]: I think to fix this for real, we should add a `*.vbproj` line here:

https://github.com/dotnet/corefx/blob/20ac3b14daf406a98a8a4966ce12c04c9d566cc0/build.proj#L65-L66

And then run `.\build-managed.cmd -- /t:UpdateVSConfigurations`, which will fix this project.  Then going forward, it won't get out of date again.

**Comment by eerhardt** [comment]
> That is caused by not having a netstandard configuration, which seems to be because for .vbproj the usual pattern like `<PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard-Debug|AnyCPU'" />` isn't sufficient: you have to also (or instead?) add it to `<Configurations>`

Check out https://github.com/dotnet/project-system/blob/master/docs/configurations.md.  With the new project system, the old PropertyGroup pattern changed to the Configurations and Platforms properties.

/cc @weshaggard

**Pull request merged**